src/Grok.Net.Powershell/Grok.Net.PowerShell.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <AssemblyName>Grok.Net.PowerShell</AssemblyName>
+        <RootNamespace>GrokNet.PowerShell</RootNamespace>
+        <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
Can we use `.net6` rather than `net461`?

Reply:
I thought `net461` is needed to support Windows PowerShell 5.1 but I realized it's not. `.netstandard2.0` alone will be just fine.